appveyor.yml

```diff
@@ -45,6 +48,7 @@ test_script:
   - ctest -C Release --verbose --timeout 90 -I 0,0,0,2
   - ctest -C Release --verbose --timeout 240 -I 0,0,0,4
   #- ctest -C Release --verbose --timeout 90 -I 0,0,0,7
+  - cd ..
 
 artifacts:
-
+  - path: build/Helics-1.0.0-beta.2-win64.exe
```

Comment:
This will likely break when we change the version number, since cpack file creation is based on the CMAKE HELICS_VERSION variable.  It would be ideal if this can key off that, but I would settle for making the name here a variable that can be updated whenever the version number inside the cmake is updated

Reply:
I have the same comment, we shouldn't hardcode.

Reply:
Can the CMake script set an environment variable with the output name of the installer?

Reply:
`set( ENV{HELICS_VERSION} ${HELICS_VERSION} )`
would set an environmental variable to the helics version code

not sure how you use that in appveyor but the -win64.exe should be pretty constant

Reply:
I tried setting the environment variable using CMake, also with PARENT_SCOPE -- it doesn't seem to be setting the environment variable for the script that ran the cmake command.

Reply:
Probably the script just picks up environment variables that were present when the script was started.